selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/testrunner/JavaTestPluginTestNgTest.java

```diff
@@ -167,7 +184,14 @@ public void shouldExecuteAlltests() {
     editor.waitActive();
     editor.goToCursorPositionVisible(26, 17);
     menu.runCommand(RUN_MENU, TEST, TEST_NG_TEST_DROP_DAWN_ITEM);
-    notifications.waitExpectedMessageOnProgressPanelAndClosed("Test runner executed successfully.");
+    try {
+
+      notifications.waitExpectedMessageOnProgressPanelAndClosed(
+          "Test runner executed successfully.");
+    } catch (WebDriverException ex) {
```

Comment:
**WebDriverException** looks too general here. It would be better to catch the real exception you encountered here.